master/internal/experiment.go

```diff
@@ -165,12 +165,6 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 			MaxSlots: e.Config.Resources().MaxSlots(),
 			Handler:  ctx.Self(),
 		})
-		if err := e.setWeight(ctx, e.Config.Resources().Weight()); err != nil {
```

Comment:
are these totally not needed? would we still get the user requested priority and weight if they set it at submission time in the config?

Reply:
it didn't seem to affect my ability to set priority or weight